README.md

```diff
@@ -12,7 +12,7 @@ button.
 
 #### Build yourself
 Build this app with gradle. In the terminal, from the root of the project, enter `./gradlew build`. On Windows, just enter `gradlew build`
-in the command line from the project root. Next, move the JAR file from build/libs to your `$XP_HOME/deploy` directory. The Security.txt
+in the command line from the project root. Next, move the JAR file from build/libs to your `$XP_HOME/deploy` directory. The Security Headers
```

Comment:
Dette var ikke det nyeste forklaringen nei 😂 Jeg vet dette egentlig ikke er en del av oppgaven, men kan du endre denne linjen til å bare si at man må installere `.jar`'en fra `build/libs` på server for å bruke den? Vi trenger ikke overkomplisere dette 😅

Reply:
Yessir!